community/detectors/gitlab_cve_2021_22205/src/main/java/com/google/tsunami/plugins/detectors/cves/cve202122205/Cve202122205VulnVo.java

```diff
@@ -0,0 +1,28 @@
+package com.google.tsunami.plugins.detectors.cves.cve202122205;
+
+import java.io.Serializable;
+
+public class Cve202122205VulnVo implements Serializable {
```

Comment:
You can just move this as an inner class of your detector and this object doesn't need to be `Serializable`, i.e.:

```java
public final class Cve202122205VulnDetector implements VulnDetector {
  // ...
  private static final class Cve202122205VulnVo {
    // ...
  }
}
```